.github/workflows/ci.yml

```diff
@@ -65,3 +65,82 @@ jobs:
       - name: Upload to codecov
         run: |
           bash <(curl -s https://codecov.io/bash)
+
+  sentry:
+    runs-on: ubuntu-latest
+    timeout-minutes: 25
+    steps:
+      # Checkout codebase
+      - name: Checkout snuba
+        uses: actions/checkout@v2
+
+      # Checkout Sentry and run integration tests against latest snuba
+      - name: Checkout sentry
+        uses: actions/checkout@v2
+        with:
+          repository: getsentry/sentry
+          path: sentry
+
+      - name: Build snuba docker image
+        run: |
+          docker pull getsentry/snuba:latest || true
```

Comment:
Why do you pull the latest image before building it? What's the benefit?

I'm surprised to find out that we upload images to both Docker Hub and Google's registry. Do you know the reason?

Reply:
Docker Hub = public consumption, GCR = generally internal use builds

Reply:
As for pulling, you need to pull in order to be able to use docker layer caching. (docker Buildkit supposedly can inline it so you don't need to pull but I haven't been able to get it to work).